change/@ni-nimble-components-b9a584ff-3c6d-46d3-afde-b9af6c45ec0e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@m-akinc @atmgrifter00 
Captured the following `Common items caught in PR review` topic on avoiding memory leaks: https://github.com/ni/nimble/issues/1479#issuecomment-2344585509

Feel free to add relevant details or other patterns that we should watch for.